test/integration/scheduler/util.go

```diff
@@ -54,7 +54,8 @@ import (
 	_ "k8s.io/kubernetes/pkg/scheduler/algorithmprovider"
 	schedulerapi "k8s.io/kubernetes/pkg/scheduler/api"
 	"k8s.io/kubernetes/pkg/scheduler/factory"
-	schedulerframework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
+	schedulerframework "k8s.io/kubernetes/pkg/scheduler/framework"
+	schedulerframeworkv1alpha1 "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
```

Comment:
This is too long for an import alias, but it seems that we don't have a better name for this.